extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -40,7 +45,7 @@ public static function updateCheckUserData( RecentChange $rc ) {
 			'cuc_minor'      => $rc_minor,
 			'cuc_user'       => $rc_user,
 			// 'cuc_user_text'  => $user->isAnon() ? $rc_user_text : '', // SUS-3080 - this column is redundant
-			'cuc_actiontext' => $actionText,
+			'cuc_actiontext' => $actionTextTrim,
 			'cuc_comment'    => $rc_comment,
 			'cuc_this_oldid' => $rc_this_oldid,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$rc_this_oldid" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[project-id]: wikia:app